frontend/packages/console-app/console-extensions.json

```diff
@@ -805,6 +795,34 @@
       "prefixNamespaced": true
     }
   },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "operatorhub",
+      "section": "ecosystem",
+      "name": "%console-app~OperatorHub%",
+      "href": "/operatorhub",
+      "insertAfter": "developer-catalog"
+    },
+    "flags": {
+      "required": ["CAN_LIST_PACKAGE_MANIFEST", "CAN_LIST_OPERATOR_GROUP"]
+    }
```

Comment:
It looks like we should not move `OperatorHub` to Ecosystem. Is that in this PR scope? 
<img width="533" alt="Screenshot 2025-06-25 at 10 34 48 AM" src="https://github.com/user-attachments/assets/d065f321-f270-4212-b7d6-16e994868d19" />